src/main/java/at/hannibal2/skyhanni/features/combat/damageindicator/DamageIndicatorManager.kt

```diff
@@ -226,8 +226,30 @@ object DamageIndicatorManager {
             }
             event.drawDynamicText(location, bossName, sizeBossName, -9f, smallestDistanceVew = smallestDistanceVew)
 
+            val icons: MutableList<String> = mutableListOf()
+            if (config.shurikenIndicator && entity.getNameTagWith(3, "§b✯") != null) {
```

Comment:
any way you can use mob detection instead of those deprecated functions? if you dont find a way, at least cache it so that the getNameTagWith function doesnt get called each frame